tools/exporter/ReadMe.md

```diff
@@ -45,11 +45,13 @@ Note:
 * Up to 1000 audits will be exported each time the software checks for new audits. If more than 1000 audits exist on the SafetyCulture platform, they will be retrieved automatically in subsequent sync cycles.
 
 ### CSV Export
-* Audits that are build off a common Template will appended one after the other into the same CSV file. 
+#### Bulk Export Mode
+* Audits that are built off a common Template will be appended one after the other into the same CSV file. 
```

Comment:
Audits conducted against the same template will be appended to a single CSV file named after the template ID (or name, whatever the case is).